modules/ROOT/pages/maven-reference.adoc

```diff
@@ -77,6 +77,73 @@ When you create a new Mule project in Anypoint Studio, the required Mule reposit
 </pluginRepository>
 ----
 
+== MuleSoft Enterprise Repository
+
+To add the MuleSoft Enterprise repository to your project, you need to meet the following prerequisites:
```

Comment:
```suggestion
To add access to the enterprise repository to your project, you must obtain and install an Enterprise license, acquire access to the repos, and configure access to the repos.
```
These are **prerequisites** to the **task** of **Add** the Enterprise Repository, a topic that is not here in the original?

Are the tasks here really: If you are working outside of AP, you need to...

1. Acquire and install an Enterprise license
2. Have credentials to access the enterprise repo (this isn't automatic with the license? why would you need to ask Support for them?)
3. Configure access to repo (are we adding the repo or just adding access to the repo?)